tests/santa/test_api_views.py

```diff
@@ -35,20 +36,90 @@ def setUpTestData(cls):
+            'name': cls.file_name,
+            'path': ('/Library/Frameworks/Compressor.framework/'
+                     'Versions/A/Resources/CompressorTranscoderX.bundle/Contents/MacOS'),
+            'sha_256': cls.file_sha256,
+            'signed_by': {
+                'common_name': cls.file_cert_cn,
+                'organization': 'Apple Inc.',
+                'organizational_unit': cls.file_team_id,
+                'sha_256': cls.file_cert_sha256,
+                'valid_from': datetime.datetime(2007, 2, 23, 22, 2, 56),
+                'valid_until': datetime.datetime(2015, 1, 14, 22, 2, 56),
+                'signed_by': {
+                    'common_name': 'Apple Code Signing Certification Authority',
+                    'organization': 'Apple Inc.',
+                    'organizational_unit': 'Apple Certification Authority',
+                    'sha_256': '3afa0bf5027fd0532f436b39363a680aefd6baf7bf6a4f97f17be2937b84b150',
+                    'valid_from': datetime.datetime(2007, 2, 14, 21, 19, 19),
+                    'valid_until': datetime.datetime(2015, 2, 14, 21, 19, 19),
+                    'signed_by': {
+                        'common_name': 'Apple Root CA',
+                        'organization': 'Apple Inc.',
+                        'organizational_unit': 'Apple Certification Authority',
+                        'sha_256': 'b0b1730ecbc7ff4505142c49f1295e6eda6bcaed7e2c68c5be91b5a11001f024',
+                        'valid_from': datetime.datetime(2006, 4, 25, 21, 40, 36),
+                        'valid_until': datetime.datetime(2035, 2, 9, 21, 40, 36),
+                    },
+                },
+            }
+        })
+        cls.file_cert = Certificate.objects.commit({
+            "organization": "Awesome Inc",
+            "common_name": f"Developer ID Application: Awesome Inc ({cls.file_team_id})",
+            "organizational_unit": f"{cls.file_team_id}",
+            "sha_256": cls.file_cert_sha256,
+            'valid_from': datetime.datetime(2006, 4, 25, 21, 40, 36),
+            'valid_until': datetime.datetime(2035, 2, 9, 21, 40, 36),
+            'signed_by': {
+                'common_name': 'Apple Root CA',
+                'organization': 'Apple Inc.',
+                'organizational_unit': 'Apple Certification Authority',
+                'sha_256': 'b0b1730ecbc7ff4505142c49f1295e6eda6bcaed7e2c68c5be91b5a11001f024',
+                'valid_from': datetime.datetime(2006, 4, 25, 21, 40, 36),
+                'valid_until': datetime.datetime(2035, 2, 9, 21, 40, 36),
+            },
+        })
+
+        cls.file_target = Target.objects.create(type=Target.BINARY, identifier=cls.file_sha256)
+        cls.file_cert_target = Target.objects.create(type=Target.CERTIFICATE, identifier=cls.file_cert_sha256)
+        cls.file_bundle_target = Target.objects.create(type=Target.BUNDLE, identifier=cls.file.bundle.bundle_id)
```

Comment:
Identifiers for BUNDLE targets are sha256 hexdigests

Reply:
my bad, don't know what I was thinking here. sorry.